test/parallel/test-shadow-realm.js

```diff
@@ -0,0 +1,12 @@
+// Flags: --harmony-shadow-realm
```

Comment:
Can we test with the new documented flag instead?

Reply:
For reference, using V8 flags in our tests has been a pain point when trying to remove those flags on V8 side (e.g. see https://github.com/nodejs/node/pull/37671)

Reply:
Thank you for sharing the context. Updated :)